install.php

```diff
@@ -18,6 +18,9 @@
  * Boston, MA  02110-1301, USA
  *
  */
+
+error_reporting(0);
```

Comment:
Why switch off the errors ?

Reply:
Otherwise, it does not redirect to installerUI.php in Nginx environment. If it not good, I can turn off only warnings.